src/AppMenuModel.cc

```
@@ -351,5 +357,68 @@ void AppMenuModel::processNext()
+        return;
+    }
+
+    QAction *cmdBarAction = findKCommandBarAction(m_menu.data());
+    if (cmdBarAction) {
+        cmdBarAction->trigger();
+    } else {
+        qCWarning(category) << "KCommandBar action not found in application DBusMenu.";
+    }
+}
+
+static QAction *findKCommandBarActionRecursive(QMenu *menu, QSet<QMenu *> &visited)
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)